datalad/metadata/search.py

```diff
@@ -107,6 +107,7 @@ def _meta2autofield_dict(meta, val2str=True, schema=None, consider_ucn=True):
                     # tailored data
                     continue
                 srcmeta[uk] = _listdict2dictlist(umeta[uk]) if umeta[uk] is not None else None
+            meta[src] = srcmeta
```

Comment:
Please review/contribute/merge to https://github.com/datalad/datalad/pull/2724 which has an alternative fix for this which wouldn't conflict but duplicate the fix